guides/README.md

```diff
@@ -61,6 +61,8 @@ Hide or show specific blocks, paragraphs, warnings or chapters via special varia
 
 When doing review, consider checking out the topic branch and putting necessary changes on top of author's work to making many comments on github.
```

Comment:
This makes no sense when I read it.

Reply:
The purpose of this PR is to define the considerations between Foreman and Satellite. We don't need to worry about this file for now.

Reply:
But the PR does not change this line. Not following? However, the sidegoal of this PR is to find and set some ground rules for work on other docs.